clients/cobol-lsp-vscode-extension/src/__tests__/ZoweApiTest.ts

```diff
@@ -12,25 +12,131 @@
  *   Broadcom, Inc. - initial API and implementation
  */
 
-import { BasicProfileManager, Session } from "@zowe/imperative";
+import { RestClient } from "@zowe/imperative";
+import { Session } from "inspector";
+import { homedir } from "os";
+import * as path from "path";
 import { ZoweApi } from "../services/ZoweApi";
 import { Type, ZoweError } from "../services/ZoweError";
-jest.mock("@zowe/imperative");
+
+const loadMock = jest.fn();
+const loadAllMock = jest.fn();
+jest.mock("@zowe/imperative/lib/profiles/src/BasicProfileManager", () => {
+    return {
+        BasicProfileManager: jest.fn().mockImplementation(() => {
+            return {
+                load: loadMock,
+                loadAll: loadAllMock,
+                getDefaultProfileName: () => "defaultName",
+            };
+        }),
+    };
+});
 
 beforeEach(() => {
-    (BasicProfileManager as any).mockClear();
+    loadMock.mockClear();
```

Comment:
jest.clearAllMocks()   can be used instead of all

Reply:
Oh, you are right. I'll fix it.